src/components/Carousel/CarouselElements/ArrowSquarePrimary/index.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { FC, ReactNode } from "react";
+import styled from "styled-components";
+
+interface ButtonProps {
+  onClick: () => void;
+  children: ReactNode;
+}
```

Comment:
consider moving this interface to types for reusing in ArrowSquareGrayOpacity/index.tsx etc.

Reply:
+